bundles/org.openhab.binding.somfytahoma/src/main/resources/OH-INF/thing/noisesensor.xml

```diff
@@ -9,6 +9,7 @@
 			<bridge-type-ref id="bridge"/>
 		</supported-bridge-type-refs>
 		<label>Somfy Noise Sensor</label>
+		<semantic-equipment-tag>Sensor</semantic-equipment-tag>
```

Comment:
Thi sis fine, but i would also agree to add an `AcousticSensor` Have seen a similar device in the ather PR M-R

Reply:
I will leave as sensor for the time being..